tests/test_user.py

```diff
@@ -973,15 +976,16 @@ def test_subject_relationships(self):
 
     def test_set_relationships(self):
         other_user = self.add_user(username='other@foo.com')
-        data = {'relationships':[{'user': TEST_USER_ID,
-                                  'has the relationship': 'partner',
-                                  'with': other_user.id},]
-               }
+        data = {'relationships': [{
+            'user': TEST_USER_ID,
+            'has the relationship': 'partner',
+            'with': other_user.id}, ]
+                }
```

Comment:
Can you fix the indentation (align with `data`), or combine it with `]` ie `'with': other_user.id}, ]}`?